python/docs/source/guide/quickstart.md

```diff
@@ -14,10 +14,10 @@ This shows the bare minimum needed to get started with Kaskada.
 ## Install
 
 Install the latest version.
-This uses `kaskdaa>=0.6.0-a.0` to ensure the pre-release version is installed.
+This uses `kaskada>=0.6.0-a.3` to ensure the pre-release version is installed.
```

Comment:
Not sure we want to get into the habit of changing these all the time. Should probably either make it a variable, or accept that `>=0.6.0-a.0` will ensure the latest version is installed.